commission/models/commission_mixin.py

```diff
@@ -50,6 +50,11 @@ def _compute_agent_ids(self):
         """Empty method that needs to be implemented in children models."""
         raise NotImplementedError()
 
+    @api.depends("product_id")
+    def _compute_commission_free(self):
```

Comment:
It's advisable to comment/docstring decisions that are difficult to understand just reading code:

```suggestion
    def _compute_commission_free(self):
        """Put as computed instead of a simple related
        to have a proper initialized value.
        """
```

Reply:
@pedrobaeza Done 🫡